doc/tutorials/using_celadon_as_uos.rst

```diff
@@ -0,0 +1,113 @@
+  to identify the USB 3.0 port header on the main board.
+* Two HDMI monitors
+
+Build Celadon from source
+*************************
+
+#. Follow the `Build Celadon from source instructions
+   <https://01.org/projectceladon/documentation/getting_started/build-source>`_
+   to get the Celadon project source code ready.
+
+#. Select Celadon build target::
+
+      $ cd <Celadon project directory>
+      $ source build/envsetup.sh
+      $ lunch cel_apl-userdebug
+
+   .. note:: You can run ``lunch`` with no arguments to manually choose your Celadon build variants.
+
+#. Download these additional patches and apply each one individually with the command::
+
+       $ git apply <patch-filename>
+
+   .. table:: ACRN patch list
+      :widths: auto
+      :name: ACRN patch list
+
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | Patch link                                                         | Description                               |
+      +====================================================================+===========================================+
+      | https://github.com/projectceladon/device-androidia/pull/458        | kernel config: Add the support of ACRN    |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia-mixins/pull/293 | graphic/mesa: Add the support of ACRN     |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia/pull/441        | cel_apl: use ttyS0 instead of ttyUSB0     |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia/pull/439        | Disable trusty and pstore                 |
+      +--------------------------------------------------------------------+-------------------------------------------+
+
+   .. note:: If the ``git apply`` command shows an error, you may need to modify
+      the source code manually instead.
+
+#. Build Celadon image::
+
+   $ device/intel/mixins/mixin-update
+   $ make SPARSE_IMG=true gptimage -j $(nproc)
+
+.. note:: Replace the ``$(nproc)`` argument with the number of processor threads on your workstation
+   in order to build the source code with parallel tasks. The Celadon gptimage will be
+   generated to ``out/target/product/cel_apl/cel_apl_gptimage.img``
+
+Steps for Using Celadon as User OS
+**********************************
+
+#. Follow :ref:`getting-started-apl-nuc` to boot the "ACRN Service OS"
+
+#. Download the `OVMF.fd firmware image
+   <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/OVMF.fd>`_
```

Comment:
Can you not use the `OVMF.fd` that's already in the repo instead? https://github.com/projectacrn/acrn-hypervisor/tree/master/devicemodel/bios

Reply:
Yes, I verified that ``OVMF.fd`` is also work for Celadon UOS, so I will update this part in next release.